src/features/PassSurvey/ui/SurveyHeader.tsx

```diff
@@ -25,6 +26,12 @@ const SurveyHeader = (props: Props) => {
     return str.length > length ? `${str.substring(0, length)}...` : str;
   };
 
+  const title = props.title ?? context.activity.name;
+
+  const isProgressDefined = props.progress !== undefined;
+
+  const titleMarginBottom = greaterThanSM ? '8px' : '10px';
+
   return (
     <Box
       paddingX={greaterThanSM ? '24px' : '16px'}
```

Comment:
Please apply paddings in the outer (upper level) component. For instance, in SurveyLayout

Reply:
I can`t do it. These paddings is applied to the header specifically.